index.html

```diff
@@ -0,0 +1,245 @@
+
+    <div class="my-projects">
+      <div class="my-project-images-container">
+        <img
+          src="../images/background.jpg"
+          alt="project image"
+          class="my-project-image"
+        />
+      </div>
+
+      <div class="my-project-info-1">
+        <h2 class="my-project-name">Multi-Post Stories</h2>
+        <p class="my-project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+
+      <div class="project-technologies">
+        <ul>
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+      </div>
+
+      <div>
+        <button type="button" class="my-project-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="my-projects">
+      <div class="my-project-image-container">
+        <img
+          src="../images/background.jpg"
+          alt="project image"
+          class="my-project-image"
+        />
+      </div>
+
+      <div class="my-project-info-2">
+        <h2 class="my-project-name">Multi-Post Stories</h2>
+        <p class="my-project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+
+      <div class="project-technologies">
+        <ul>
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+      </div>
+
+      <div>
+        <button type="button" class="my-project-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="my-projects">
+      <div class="my-project-image-container">
+        <img
+          src="../images/background.jpg"
+          alt="project image"
+          class="my-project-image"
+        />
+      </div>
+
+      <div class="my-project-info-3">
+        <h2 class="my-project-name">Multi-Post Stories</h2>
+        <p class="my-project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+
+      <div class="project-technologies">
+        <ul>
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+      </div>
+
+      <div>
+        <button type="button" class="my-project-button">See Project</button>
+      </div>
+    </div>    
+
+    <div class="my-projects">
+      <div class="my-project-image-container">
+        <img
+          src="../images/background.jpg"
```

Comment:
The same bug requested in the comment below, Please, fix the image problem by changing the `src` to `src="./images/background.jpg"`

Reply:
Fixed also... thanks.